packages/apps/human-app/frontend/src/modules/worker/jobs/hooks/use-filter-updates.ts

```diff
@@ -0,0 +1,23 @@
+import { useJobsFilterStore } from './use-jobs-filter-store';
+
+interface UseFilterUpdatesOptions {
+  isMobile?: boolean;
+}
+
+export function useFilterUpdates({
+  isMobile = false,
+}: UseFilterUpdatesOptions = {}) {
+  const { setFilterParams, filterParams } = useJobsFilterStore();
+
+  const updateFilterParams = (updates: Partial<typeof filterParams>) => {
+    setFilterParams({
+      ...updates,
+      ...(isMobile ? { page: 0 } : {}),
```

Comment:
`setFilterParams` will always set `page: 0`, so I'm not sure we need this `useFilterUpdates` hook at all